lib/bolt/pal.rb

```diff
@@ -39,6 +42,8 @@ def self.load_puppet
         raise Bolt::CLIError, "Puppet must be installed to execute tasks"
       end
 
+      require 'bolt/pal/logging'
```

Comment:
This is an odd pattern. Seems like it'd usually be done with a Module that's loaded at the top, then used here.